connectors/tests/test_utils.py

```diff
@@ -58,18 +58,28 @@ def test_invalid_names():
 @pytest.mark.asyncio
 async def test_mem_queue_speed(patch_logger):
     # with memqueue
-    queue = MemQueue(maxmemsize=1024 * 1024, refresh_interval=0.1, refresh_timeout=2)
-    start = time.time()
-    for i in range(1000):
-        await queue.put("x" * 100)
-    mem_queue_duration = time.time() - start
+    mem_queue_duration = 10000
```

Comment:
Should this change be in this PR actually? Seems irrelevant.

Reply:
Which discussion with Tim?

Reply:
This test was flaky and sometimes broke locally and in the CI. I proposed to reduce the probability of it breaking and use `timeit` instead of `time` as `timeit` is designed for these kinds of measurements (f.e. turns of GC etc.). But this change also reduces the probability of breaking randomly a lot, so lgtm.

Reply:
yeah it's similar to timeit -- I'll put it in a separate patch